pkg/controller/migmigration/hooks.go

```diff
@@ -79,6 +91,9 @@ func (t *Task) ensureJob(job *batchv1.Job, hook migapi.MigPlanHook, migHook miga
 	} else if runningJob.Status.Failed >= HookJobFailedLimit {
 		err := fmt.Errorf("Hook job %s failed.", runningJob.Name)
 		return false, err
+	} else if len(runningJob.Status.Conditions) > 0 && runningJob.Status.Conditions[0].Reason == BackoffLimitExceededError {
```

Comment:
I think `HasCondition` might be a good fit for this? https://github.com/konveyor/mig-controller/blob/master/pkg/apis/migration/v1alpha1/condition.go#L249

Reply:
Let me test and change it.

Reply:
I'm actually not clear on how to implement this. Do I have to create a HasConditions function specifically for jobs?

Reply:
You're right, I don't think we want to implement this for jobs. Marking as resolved.